### ISSUE TITLE
[Feature Request] Limit tag view to top 20

### ISSUE THREAD

**Issue by michel-thomas** [jendrikseipp/rednotebook#385]
### Versions
RedNotebook version: 2.3
Operating system and version: Ubuntu 17.10

### Expected behavior
Ability to limit the number of tags in left view, for example only keep the 20 most used tags (number could be changed in config).

### Actual behavior
All tags appears even if used only once, with different character size, and make it difficult to read.

### Steps to reproduce the behavior
Having lots of tags by placing # behind keywords you want to highlight.

**Comment by jendrikseipp** [comment]
I agree, this feature would be nice to have. If anyone is up to the task, pull requests are welcome.

**Comment by jendrikseipp** [comment]
Implemented in #456
